gallery/src/util/jsUtil.ts

```diff
@@ -12,3 +12,32 @@
 export function ensureArray<T>(items: T | T[]): T[] {
   return Array.isArray(items) ? items : [items];
 }
+
+// Quick and dirty removal of any comment appearing at the beginning of the content, as it is assumed to be
+// a license which is not helpful to the example display. Both HTML-syntax and C-syntax comments are removed
+export const removeLicense = (content: string) =>
```

Comment:
I'd be in favor of these being in a separate utility file - `exampleUtil.ts` or something.  I typically think of `jsUtil` files as being for functions that are very general to JS needs and not to something app-specific.

Reply:
> I'd be in favor of these being in a separate utility file - `exampleUtil.ts` or something. I typically think of `jsUtil` files as being for functions that are very general to JS needs and not to something app-specific.

The files have now been split up as suggested: 78d0064b85aec5a12d11497eeafcfa1b5ea14474